src/main/java/org/netcracker/educationcenter/elasticsearch/Connection.java

```diff
@@ -0,0 +1,103 @@
+package org.netcracker.educationcenter.elasticsearch;
+
+import org.apache.http.HttpHost;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * This class establishes a connection with Elasticsearch Database using rest client
+ * to perform requests based on the provided builder.
+ *
+ * @author Mikhail Savin
+ */
+public class Connection implements AutoCloseable {
+    private static final Logger LOG = LogManager.getLogger();
+
+    /**
+     * High-level client instance to establish connection
+     */
+    private RestHighLevelClient restHighLevelClient;
+
+    /**
+     * Elasticsearch hostname
+     */
+    private String hostname;
+
+    /**
+     * The name of the scheme.
+     */
+    private String scheme;
+
+    /**
+     * First port number.
+     */
+    private int port1;
+
+    /**
+     * Second port number
+     */
+    private int port2;
+
+    /**
+     * A constructor which is used to load properties and make a connection with Elasticsearch
+     */
+    public Connection() {
+        loadProperties();
+    }
+
+    /**
+     * @return High-level client instance
+     */
+    public RestHighLevelClient getRestHighLevelClient() {
+        return restHighLevelClient;
+    }
+
+    /**
+     * This method loads properties from connection.properties and initializes property fields such as
+     * hostname, scheme, port1 and port2
+     */
+    private void loadProperties() {
+        Properties properties = new Properties();
+
+        try (FileInputStream fileInputStream = new FileInputStream("src/main/resources/connection.properties")) {
```

Comment:
This won't work in production because you will not have this folder on server.
To get file from resources use 'getClass().getResourceAsStream()'